.github/workflows/test-code.yml

```diff
@@ -51,7 +51,6 @@ jobs:
         if: ${{ startsWith(matrix.os, 'ubuntu') }}
         working-directory: src
         run: |
-          dotnet tool install -g dotnet-format --version 6.0.243104 --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
```

Comment:
You may also revert 6.0.102 to 6.0.x in all other workflows.

Reply:
So 6.0.x works properly as long as we don't try to install the dotnet-format tool (because it is already included) ?

Reply:
Yes. Also see this https://github.com/dotnet/format/issues/1540

Reply:
> So 6.0.x works properly as long as we don't try to install the dotnet-format tool (because it is already included) ?

Sorry. I misunderstood your question.

Reply:
It does ot matter we use `dotnet-fromat` or `dotnet format`. It always fails on 6.0.200. Either version works on 6.0.102.

Reply:
So I see...